diff -urN ocfs2-tools-1.4.4.orig//vendor/common/ocfs2.init ocfs2-tools-1.4.4/vendor/common/ocfs2.init
--- ocfs2-tools-1.4.4.orig//vendor/common/ocfs2.init	2008-12-03 19:38:23.000000000 +0100
+++ ocfs2-tools-1.4.4/vendor/common/ocfs2.init	2010-06-24 11:26:49.000000000 +0200
@@ -7,12 +7,12 @@
 #
 ### BEGIN INIT INFO
 # Provides: ocfs2
-# Required-Start: $network o2cb
-# Required-Stop: 
+# Required-Start: $local_fs $network o2cb
+# Required-Stop: $local_fs
 # X-UnitedLinux-Should-Start:
 # X-UnitedLinux-Should-Stop:
-# Default-Start:  2 3 5
-# Default-Stop:
+# Default-Start: S
+# Default-Stop: 0 6
 # Short-Description: Mount OCFS2 volumes at boot.
 # Description:  Mount OCFS2 volumes at boot.
 ### END INIT INFO
